File: app/src/main/java/com/wealoha/social/beans/TopicPostDTO.java
package com.wealoha.social.beans;

import java.util.List;

public class TopicPostDTO {

	public String postId;
	public FeedType type;
	public String description;
	public long createTimeMillis;
	public boolean mine;
	public String venue;
	public String venueId;
	public Double latitude;
	public Double longitude;
	public Boolean venueAbroad;
	public UserDTO user;
	public List<TopicPostTagDTO> userTags;
	public ImageCommonDto image;
	public VideoCommonDTO video;
	public boolean tagMe;
	public boolean liked;
	public int commentCount;
	public int praiseCount;
	public List<Comment2DTO> recentComment;
	public HashTagDTO hashtag;
	public boolean hasMoreComment;
}

File: app/src/main/java/com/wealoha/social/beans/TopicPostDTO.kt
package com.wealoha.social.beans

data class TopicPostDTO(

    var postId: String,

    var type: FeedType,

    var description: String = "",

    var createTimeMillis: Long = 0,

    var mine: Boolean = false,

    var venue: String? = null,

    var venueId: String? = null,

    var latitude: Double? = null,

    var longitude: Double? = null,

    var venueAbroad: Boolean? = null,

    var user: UserDTO,

    var userTags: List<TopicPostTagDTO>? = null,

    val image: ImageCommonDto,

    val video: VideoCommonDTO,

    var tagMe: Boolean = false,

    var liked: Boolean = false,

    var commentCount: Int = 0,

    var praiseCount: Int = 0,

    var recentComment: List<Comment2DTO>? = null,

    var hashtag: HashTagDTO? = null,

    var hasMoreComment: Boolean = false
) {

}